packages/react/src/prefabs/VideoConference.tsx

```diff
@@ -9,8 +9,11 @@ import { TileLoop } from '../components/TileLoop';
 import { Chat } from './Chat';
 import { ConnectionStateToast } from '../components/Toast';
 import { useMediaQuery } from '../hooks/utiltity-hooks';
+import { MessageFormatter } from './ChatEntry';
 
-export type VideoConferenceProps = React.HTMLAttributes<HTMLDivElement>;
+export interface VideoConferenceProps extends React.HTMLAttributes<HTMLDivElement> {
+  formatChatMessage?: MessageFormatter;
```

Comment:
`formatChatMessage` could also be the name of a boealean variable. What do you think of `chatMessageFormatter` to avoid ambiguity?